index.js

```diff
@@ -1,4 +1,11 @@
 const functions = require('firebase-functions')
 const app = require('./src/app')
+const onTrackUrlChange = require('./src/on-track-url-change')
 
+// Public API for Radio4000
 exports.api = functions.https.onRequest(app)
+
+// Listen to every track change
+exports.onTrackUrlChange =
+	functions.database.ref('/tracks/{trackId}/url')
+	.onWrite(onTrackUrlChange)
```

Comment:
does `onWrite` mean create+update? Does it also include delete?

Reply:
> onWrite(), which triggers when data is created, updated, or deleted in the Realtime Database.
Source: https://firebase.google.com/docs/functions/database-events